runtime/browser/native_window.cc

```diff
@@ -216,4 +218,13 @@ void NativeWindow::FullScreen(bool enable) {
       enable ? ELM_WIN_INDICATOR_TRANSPARENT : ELM_WIN_INDICATOR_OPAQUE);
 }
 
+NativeWindow::WindowCategory NativeWindow::GetWindowCategory(
```

Comment:
I can't found declared of this function

I can found only 
WebApplication 
NativeWindow::WindowCategory GetWindowCategory(common::AppControl* ac);
in web_application.h

Fix it